### PR TITLE
fix for fetching system messages file on Redhat/CentOS distribution with no lsb_release

### PR DESCRIPTION
**Pull request by hhakimi53**

### CODE REVIEW

File: src/frontend/org/voltdb/utils/Collector.java [hunk 1]
```diff
@@ -277,6 +280,37 @@ private static void parseJSONObject(JSONObject jsonObject) {
         }
     }
 
+    private static String getLinuxOSInfo() {
+        // Supported Linux OS for voltdb are CentOS, Redhat and Ubuntu
+        String versionInfo = "";
+        try {
```
Comment (akhanzode) [inline src/frontend/org/voltdb/utils/Collector.java]: Use try with resource

File: src/frontend/org/voltdb/utils/Collector.java [hunk 2]
```diff
@@ -277,6 +280,37 @@ private static void parseJSONObject(JSONObject jsonObject) {
...
+            BufferedReader br = null;
+            // base Ubuntu distributions have the distribution info in /etc/lsb-release
+            final String ubuntuDistInfoFilePath = "/etc/lsb-release";
+            // base RedHat, CentOS distributions have the distribution info in /etc/redhat-release
+            final String rhDistInfoFilePath = "/etc/redhat-release";
```
Comment (akhanzode) [inline src/frontend/org/voltdb/utils/Collector.java]: Make this into a list so you can iterate till you find one. This way we can add to list.

File: src/frontend/org/voltdb/utils/Collector.java [hunk 3]
```diff
@@ -326,19 +360,23 @@ private static void parseJSONObject(JSONObject jsonObject) {
             }
 
             String systemLogBase;
+            final String systemLogBaseDirPath = "/var/log/";
```
Comment (akhanzode) [inline src/frontend/org/voltdb/utils/Collector.java]: This is only for Linux right so can be in else.

File: src/frontend/org/voltdb/utils/Collector.java (hunk 2 above)
Comment (hhakimi53) [inline src/frontend/org/voltdb/utils/Collector.java]: Sure.

File: src/frontend/org/voltdb/utils/Collector.java (hunk 1 above)
Comment (hhakimi53) [inline src/frontend/org/voltdb/utils/Collector.java]: Sure

File: src/frontend/org/voltdb/utils/Collector.java (hunk 3 above)
Comment (hhakimi53) [inline src/frontend/org/voltdb/utils/Collector.java]: This is for both Mac and Linux.

**Pull request merged**